garrysmod/gamemodes/terrortown/gamemode/vgui/sb_row.lua

```diff
@@ -408,9 +408,16 @@ function PANEL:ShowMicVolumeSlider()
          )
          draw.DrawText(textValue, "cool_small", sliderHeight / 2, -20, Color(255, 255, 255, 255), TEXT_ALIGN_CENTER)
       end
-      
+
       draw.RoundedBox(100, 0, 0, sliderHeight, sliderHeight, Color(255, 255, 255, 255))
    end
 end
 
+local function HideVolume()

```

Comment:
If you move this above `PANEL:ShowMicVolumeSlider`, you can use it inside the timer as well to reduce code duplication.